raiden/tests/fixtures/blockchain.py

```diff
@@ -37,14 +38,15 @@
     ('deploy_service', 'blockchain_services'),
 )
 log = slogging.getLogger(__name__)  # pylint: disable=invalid-name
+DAGSIZE = 1073739912
```

Comment:
Maybe add a comment here or renamed the variable to `EPOCH0_DAGSIZE`, just in case anyone in the future sees it and misunderstands that we always think the DAG has the same size for all epochs.